parlai/agents/bert_ranker/bert_dictionary.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+from parlai.core.dict import DictionaryAgent
+try:
+    from pytorch_pretrained_bert import BertTokenizer
+except ImportError:
+    raise Exception(("BERT rankers needs pytorch-pretrained-BERT installed. \n "
```

Comment:
make this an ImportError too, to be consistent with TA and other places in ParlAI